src/rust/engine/logging/src/logger.rs

```diff
@@ -164,8 +169,24 @@ impl Log for Logger {
           cur_date.format(TIME_FORMAT_STR),
           cur_date.time().nanosecond() / 10_000_000 // two decimal places of precision
         );
+
         let level = record.level();
-        let log_string: String = format!("{} [{}] {}", time_str, level, record.args());
+        let use_color = self.use_color.load(Ordering::SeqCst);
+
+        let level_marker = match level {
+          _ if !use_color => format!("[{}]", level).normal().clear(),
+          Level::Info => format!("[{}]", level).normal().bold(),
+          Level::Error | Level::Warn => format!("[{}]", level).red().bold(),
```

Comment:
WARN in yellow is fairly standard... that would probably preferable unless there is guidance somewhere saying that yellow is hard for people to see.

Reply:
Yellow looks kind of bad on a light-colored background, so I'm reluctant to use it. I tried displaying yellow on a black background, but I don't like how that looks either:

![screenshot](https://user-images.githubusercontent.com/29717477/87491384-d7224800-c5fc-11ea-875e-259b8ef4fe7a.png)

I also got rid of the red text for error messages only on @Eric-Arellano 's suggestion, but hten we have no way of distinguishing WARN and ERROR messages. Maybe that's fine, though, since they're both exceptional cases, and right now pants isn't really making much systematic use of the distinction.

Reply:
Whichever yellow pytest is using there is a better one I think? Slightly darker?

Reply:
> Whichever yellow pytest is using there is a better one I think? Slightly darker?

That's not yellow that pytest is using, that's a light green.

Reply:
<img width="290" alt="Screen Shot 2020-07-15 at 10 59 11 AM" src="https://user-images.githubusercontent.com/46740/87579081-3c704a80-c68a-11ea-9149-26fc15244717.png">
^ Whatever it's using here, it's a better "yellow".

Reply:
> <img alt="Screen Shot 2020-07-15 at 10 59 11 AM" width="290" src="https://user-images.githubusercontent.com/46740/87579081-3c704a80-c68a-11ea-9149-26fc15244717.png">
> ^ Whatever it's using here is a better "yellow".

I'm pretty sure that's the same yellow, and I find it equally hard to read

Reply:
Changed this to use unbolded red for both ERROR and WARN (and removed bolding from INFO)

Reply:
Hm. It's definitely not the same (checked with a color picker). But if you're not able to read it then I guess that's that.

Reply:
To be 100% clear: I'm talking about the "1 skipped" text here being in a darker shade of yellow than the "v3.8.3".